spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/ConfigMapsFromFilePathsTests.java

```diff
@@ -65,17 +65,13 @@
 	protected static final String FIRST_FILE_NAME_DUPLICATED_FULL_PATH = FILES_ROOT_PATH + "/" + FILES_SUB_PATH + "/"
 			+ FIRST_FILE_NAME;
 
-	@ClassRule
```

Comment:
this is now injected by `@ExtendWith(SpringExtension` so no need for the server. the rest of the changes are trivial, imho, here.